README.md

```diff
@@ -2,6 +2,79 @@
 
 This project contains Kubernetes manifests for self-deployed OTLP ingest on Kubernetes.
 
+## Running on GKE
+
+The recommended way to add the collector to your deployment is using `kubectl
+apply`. If running on a GKE Autopilot cluster (or any cluster with Workload
+Identity), you must follow the prerequisite steps to set up a Workload
+Identity-enabled service account below. Otherwise, you can skip to the next
+section.
+
+### Workload Identity prequisites
+
+Follow the [Workload Identity
+docs](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#authenticating_to)
+to set up an IAM service account in your GCP project with permission to use
+Workload Identity and write logs, traces, and metrics:
+
+```console
+export GCLOUD_PROJECT=<your project id>
+export NAMESPACE=opentelemetry-demo # Enter your application namespace
+```
+
+Note: If you don't have a namespace yet you can create one and use it as current
+context like this:
+```console
+kubectl create namespace ${NAMESPACE}
+kubectl config set-context --current --namespace ${NAMESPACE}
+```
+Then run the following to create the service account with the appropriate permissions:
+```console
+gcloud iam service-accounts create opentelemetry-collector \
+    --project=${GCLOUD_PROJECT}
+gcloud projects add-iam-policy-binding ${GCLOUD_PROJECT} \
+    --member "serviceAccount:opentelemetry-collector@${GCLOUD_PROJECT}.iam.gserviceaccount.com" \
+    --role "roles/logging.logWriter"
+gcloud projects add-iam-policy-binding ${GCLOUD_PROJECT} \
+    --member "serviceAccount:opentelemetry-collector@${GCLOUD_PROJECT}.iam.gserviceaccount.com" \
+    --role "roles/monitoring.metricWriter"
+gcloud projects add-iam-policy-binding ${GCLOUD_PROJECT} \
+    --member "serviceAccount:opentelemetry-collector@${GCLOUD_PROJECT}.iam.gserviceaccount.com" \
+    --role "roles/cloudtrace.agent"
+gcloud iam service-accounts add-iam-policy-binding opentelemetry-collector@${GCLOUD_PROJECT}.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:${GCLOUD_PROJECT}.svc.id.goog[${NAMESPACE}/opentelemetry-collector]"
+```
+
+### Install the manifests
+
+First, make sure you have followed the Workload Identity setup steps above.
+Update [`rbac.yaml`](rbac.yaml) to annotate the Kubernetes service account with
+your project:
+
+
+```console
+sed -i "s/%GCLOUD_PROJECT%/${GCLOUD_PROJECT}/g" collector/2_rbac.yaml
```

Comment:
curious would this work as just something like 

```
sed -i "s/%GCLOUD_PROJECT%/${GCLOUD_PROJECT}/g" collector/*
```

to apply to all the install manifests? looks a little cleaner too. Numbering the manifests is fine, but it looks a little weird as a user when you need to make a change like this to one of them

Reply:
Good point, it does work. Updated the readme